pallets/author-mapping/src/lib.rs

```diff
@@ -96,16 +117,18 @@ pub mod pallet {
 		AuthorRegistered {
 			author_id: NimbusId,
 			account_id: T::AccountId,
+			keys: T::Keys,
 		},
 		/// An NimbusId has been de-registered, and its AccountId mapping removed.
-		AuthorDeRegistered { author_id: NimbusId },
+		AuthorDeRegistered { author_id: NimbusId, keys: T::Keys },
```

Comment:
I think this event should include the `AccountId` too.

Reply:
done